scripts/run-generated-project-smoke.mjs

```
@@ -453,8 +455,21 @@ function main() {
+		throw error;
 	} finally {
-		fs.rmSync(tempRoot, { force: true, recursive: true });
+		try {
+			cleanupTemporaryProjectRoot(tempRoot);
+		} catch (cleanupError) {
+			if (primaryError) {
+				console.warn(
+					`Warning: failed to remove temporary smoke directory ${tempRoot}: ${cleanupError instanceof Error ? cleanupError.message : String(cleanupError)}`,
+				);
+			} else {
+				throw cleanupError;
```

Biome (2.4.11)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)